Expensify/react-native-google-places-autocomplete
# Update latest version

### Details
This update is requested in [this issue](https://github.com/Expensify/App/issues/16448). This PR only merge the latest version 2.5.1 from the original react-native-google-places-autocomplete library.
### Fixed Issues

$ https://github.com/Expensify/App/issues/16448
PROPOSAL: https://github.com/Expensify/App/issues/16448#issuecomment-1492881279


### Tests

1. Update commit hash `e8346780c43c33b55b62d0241d8a209b48c522a5` for react-native-google-places-autocomplete in package.json then run npm install 
2. Open the app
3. Open Workspaces > connect bank account
4. Click on connect manually
5. Fill in step 1 information (eg data: routing number = 210021002, acc no = 1234)
6. In the company address, type in address but don't select any option
7. Click outside (ex: click on the city field)
8. Verify that the suggestion text below the company address is displayed


### Offline tests


### QA Steps

1. Update commit hash `e8346780c43c33b55b62d0241d8a209b48c522a5` for react-native-google-places-autocomplete in package.json then run npm install 
2. Open the app
3. Open Workspaces > connect bank account
4. Click on connect manually
5. Fill in step 1 information (eg data: routing number = 210021002, acc no = 1234)
6. In the company address, type in address but don't select any option
7. Click outside (ex: click on the city field)
8. Verify that the suggestion text below the company address is displayed



### Screenshots/Videos
<details>
<summary>Web</summary>

https://user-images.githubusercontent.com/113963320/229745977-d3f771ff-d6f1-4326-bbfb-0334b9aa6760.mov


</details>




<details>
<summary>Mobile Web - Chrome</summary>


https://user-images.githubusercontent.com/113963320/229746026-a9eb0b86-88d9-4455-952f-ea2a6c3e0678.mov


</details>

<details>
<summary>Mobile Web - Safari</summary>





https://user-images.githubusercontent.com/113963320/229746038-9a9f3820-c871-4fe6-ac1f-4938c8b52b14.mov



</details>





<details>
<summary>Desktop</summary>




https://user-images.githubusercontent.com/113963320/229746180-e636d459-00ae-4afb-b9f1-64a4a5794169.mov


</details>

<details>
<summary>IOS</summary>




https://user-images.githubusercontent.com/113963320/229827548-864d73bd-1782-455b-ac06-e03ff951fdb9.mov




</details>

<details>
<summary>Android</summary>

https://user-images.githubusercontent.com/113963320/229827187-11f8d187-3833-4b23-9afe-446e6c83fd5f.mp4





</details>
